src/rules/LinearAlgebra/blas.jl

```diff
@@ -0,0 +1,72 @@
+const ConstOrDuplicated{T} = Union{Const{T},Duplicated{T}}
+
```

Comment:
Can you also add batch duplicated for these as well?

Otherwise someone successfully using the fallback right now in vector mode will now get a runtime rule not found error?

Reply:
Is this related at all to `ConfigWidth`? i.e. if a `BatchDuplicated` is used, then will `ConfigWidth` indicate the number of shadows contained in the `BatchDuplicated`?

Reply:
Yes, all duplicated will be of the same width. Here it must be passed as such in case there are No duplicated arguments